src/Aspire.Hosting.PostgreSQL/PostgresBuilderExtensions.cs

```diff
@@ -424,6 +424,58 @@ public static IResourceBuilder<PostgresDatabaseResource> WithCreationScript(this
         return builder;
     }
 
+    /// <summary>
+    /// Configures the password that the PostgreSQL resource is used.
+    /// </summary>
+    /// <param name="builder">The resource builder.</param>
+    /// <param name="password">The parameter used to provide the password for the PostgreSQL resource.</param>
+    /// <returns>The <see cref="IResourceBuilder{T}"/>.</returns>
+    public static IResourceBuilder<PostgresServerResource> WithPassword(this IResourceBuilder<PostgresServerResource> builder, IResourceBuilder<ParameterResource> password)
+    {
+        ArgumentNullException.ThrowIfNull(builder);
+        ArgumentNullException.ThrowIfNull(password);
+
+        builder.Resource.SetPassword(password.Resource);
+        return builder.WithEnvironment(context =>
+        {
+            context.EnvironmentVariables[PasswordEnvVarName] = builder.Resource.PasswordParameter;
+        });
```

Comment:
We shouldn't need this because this code grabs the parameters from the resource, so it will continue to work if we set the parmaeter correctly.

https://github.com/dotnet/aspire/blob/fd413eb66b219013ad4b8bc64a459c84a82fa446/src/Aspire.Hosting.PostgreSQL/PostgresBuilderExtensions.cs#L110-L113

(repeated elsewhere)